src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -28,6 +28,7 @@
 #include <app/EventLogging.h>
 #include <app/server/Server.h>
 #include <app/util/af.h>
+#include <app/util/attribute-storage.h>
```

Comment:
Just realized: why do we need to include attribute-storage.h?  What are we using from there?  The answer should be "nothing" and we should not include it here....

Reply:
I think i forgot to remove this we\hen cleaning up.

Reply:
don't we need it for registering the class for attribute access overide - https://github.com/project-chip/connectedhomeip/blob/adcf7423514743c4ab693b87db83f3bbfb97789f/src/app/util/attribute-storage.h#L226